base/array.jl

```diff
@@ -1697,7 +1697,7 @@ end
 
 Returns the maximum element of the collection `itr` and its index. If there are multiple
 maximal elements, then the first one will be returned. `NaN` values are ignored, unless
-all elements are `NaN`.
+all elements are `NaN`. Except of that, the result is in line with `max`.
```

Comment:
This is a bit unclear. I'd maybe phrase this as "Other than the treatment of `NaN`, the result..."